scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+
+[Service]
+User=haas_user
+Group=haas_user
+WorkingDirectory=/var/lib/haas/
+ExecStart=/usr/bin/haas serve_networks
```

Comment:
I'd prefer to get rid of the `KillMode` line, and let it [default to control-group](https://www.freedesktop.org/software/systemd/man/systemd.kill.html). That way, if there's an inadvertent background process, it'll get killed as well.

Also - is there a way to make haas dependent on the postgres service if it's present? We could alternatively mark this as after